<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jaaferwindows</class>
 <widget class="QMainWindow" name="jaaferwindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>jaaferwindows</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>571</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajout</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>311</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter Facture</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Id Facture</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Date </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Montant</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_montant">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ajout_button">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>220</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Methode de paiement</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_method">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="lineEdit_date">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>220</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>pdf</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Affichage</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>331</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>154</x>
        <y>270</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>trie montant</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="asc">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>260</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>asc</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="desc">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>280</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>desc</string>
      </property>
     </widget>
     <widget class="QPushButton" name="all">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher tout</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>rechere id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idsearch">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>180</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="qrcode">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>0</y>
        <width>191</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Modification</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>311</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Modifier Facture</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Id Facture</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Date </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Montant</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_montant_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier_button">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>220</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Methode de paiement</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_method_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="lineEdit_date_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>suppression</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_supprimer">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Id de la facture à supprimer:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
